charts/pxc-operator/values.yaml

```diff
@@ -27,17 +27,17 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
-resources: {}
+resources:
   # We usually recommend not to specify default resources and to leave this as a conscious
   # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+  # resources, such as Minikube. If you don't want to specify resources, comment the following
+  # lines and add the curly braces after 'resources:'.
+  limits:
+    cpu: 200m
+    memory: 100Mi
```

Comment:
We have increased this to 500Mi because it was leading to OOM on OpenShift environment with 100Mi, so please change it here also.

Reply:
Resolved in 5a7f4bfaed5157da4420c8e83f91cdf2e1e45c46